Lib/platform.py

```diff
@@ -134,7 +134,11 @@
 
 # Directory to search for configuration information on Unix.
 # Constant used by test_platform to test linux_distribution().
-_UNIXCONFDIR = '/etc'
+try:
+    _UNIXCONFDIR = '/etc/os-release'
+    open(_UNIXCONFDIR)
```

Comment:
ResourceWarning will be emitted here.

In general, performing filesystem operations at import time can be considered a bad style.

Reply:
here i define _UNIXCONFDIR because test_platform import it variable, and if os-release directory will be defined in function, test_platform finished with error.

Reply:
Wouldn't be better to search the `'os-release'` file in the `_UNIXCONFDIR` directory similarly to searching other config files?

Reply:
If search os-release file using an existing method, it is necessary to make a lot of changes to it, since the os-release file after sorting is in the last places and therefore it is not using in the future. I think that can find, if not, then look for the rest of the files. Now I have remade the commit in accordance with your remarks.